conans/client/cmd/profile.py

```diff
@@ -67,6 +67,8 @@ def cmd_profile_update(profile_name, key, value, cache_profiles_path):
         profile.options.update(tmp)
     elif first_key == "env":
         profile.env_values.update_replace(rest_key, value)
+    elif first_key == "conf":
+        profile.conf.loads("{}={}".format(rest_key, value), reset=False)
```

Comment:
wouldn't it be better to implement a ``conf.update(key, value)``? And avoid the ``reset`` arg.

Reply:
Completely agree. Doing that!